Tes avi

fahrizalrahman opened the pull request:
Perbaikan Form Edit Penjualan & Form Penjualan

- Perhitungan Form Pembayaran (Javascript)
- Submit produk (PRODUK jasa tanpa alert stok tidak mencukupi )
- Rapihkan Form Edit Penjualan
- Perbaikan Proses Edit Penjualan penambahaan diskon bertingkat

fahrizalrahman committed:
Perbaikan Form Edit Penjualan & Form Penjualan

- Perhitungan Form Pembayaran (Javascript)
- Submit produk (PRODUK jasa tanpa alert stok tidak mencukupi )
- Rapihkan Form Edit Penjualan
- Perbaikan Proses Edit Penjualan penambahaan diskon bertingkat

fahrizalrahman committed:
Merge remote-tracking branch 'origin/tes_avi' into tes_avi

RindangRamadhan commented:
REVIEW CODE FAHRIZAL (RAMA)

- Edit Penjualan, logika perhitungan hapus produk kurang tepat. Seharusnya ketika klik hapus dan tidak mgklik yes maka total tidak berubah (Form Edit Penjualan)
- $select_setting_akun = $db->query("SELECT * FROM setting_akun"); >> Tologn dibenahi, ambil yang diperlukan saja (Proses Bayar Edit Jual)

fahrizalrahman committed:
Perbaikan Review Code Rama & pergantian datatable ajax edit penjualan 

- hapus query tidak penting 
- penambahan datatable ajax
- perbaikan proses di edit penjualan

rikopernando commented on cek_stok_konversi_edit_penjualan.php:
```diff
@@ -14,12 +14,14 @@
 
 
 
+$queryy = $db->query("SELECT SUM(sisa) AS total_sisa FROM hpp_masuk WHERE kode_barang = '$kode_barang' ");
```
perhitungan hpp nya di ubah ya mas, jika tidak gunakan saja funtion perhitungan hpp

fahrizalrahman committed:
Perbaikan Testing & Review Kode 

- Ppn default non &
- query cek persediaan include dari persediaan.function.php

fahrizalrahman committed:
Perbaikan master data satuan  

- datatable eror ganti dengan datatable ajax 
- perbaikan proses edit modal edit tidak tertutup saat submit edit

RindangRamadhan committed:
Penjualan

- Ubah trigger detail_penjualan
- Menambahkan peringtan jika ada produk yang stoknya tidak mencukupi

RindangRamadhan committed:
Merge remote-tracking branch 'origin/tes_avi' into tes_avi

RindangRamadhan committed:
Lap. Penjualan Piutang Beredar

- Membenahi logika menghitung usia piutang di lap. piutang beredar

fahrizalrahman reviewed (commented):
Review Code (Rama)
    * Barcode.php
        * select * mints oolong diganti
    * file all
        * komentar query / javascrip untuk yang 
           ditambahkan

fahrizalrahman reviewed (commented):
Review Kode 
Tidak ada masalah

fahrizalrahman committed:
perbaikan edit jumlah di tbs penjualan 

- saat diedit muncul peringatan / alert jumlah tidak mencukupi (produk jasa )
- ganti datatable ajax tbs penjualan form penjualan

RindangRamadhan committed:
Perbaikan Review (Fahrizal)

- Memperbaiki select * dan menambahkan komentar

RindangRamadhan committed:
Merge remote-tracking branch 'origin/tes_avi' into tes_avi

hendriandaglos commented:
aplikasi
1. harga di beri tanda pemisah titik (tampilannya)
2. submit tbs and hapus tbs (harga kosong)
3. edit jumlah di tbs tidak boleh kosong ,
4. setelah harga kosong , dan di ubah menjadi jumlah 1 ++ pajak jadi NAN, dan subtotal dan total akhir NAN

fahrizalrahman committed:
Perbaikan Review Kode (Hendri)

 1. harga di beri tanda pemisah titik (tampilannya)
2. submit tbs and hapus tbs (harga kosong)
3. edit jumlah di tbs tidak boleh kosong ,
4. setelah harga kosong , dan di ubah menjadi jumlah 1 ++ pajak jadi NAN, dan subtotal dan total akhir NAN

fahrizalrahman committed:
Merge remote-tracking branch 'origin/tes_avi' into tes_avi
